src/content/docs/journey-migration/migrating-infra.mdx

```diff
@@ -0,0 +1,230 @@
+---
+title: "Install infrastructure agents"
+tags:
+- Learning paths
+- Customer experience
+- Datadog
+- Migration
+metaDescription: "How to migrate your infrastructure monitoring from Datadog to New Relic."
+---
+
+import journeyGuidedDocker from 'images/infrastructure_screenshot-crop_guided-docker.png'
+
+import journeyGuidedK8 from 'images/infrastructure_screenshot-crop_guided-kubernetes.png'
+
+import journeyGuidedLinux from 'images/infrastructure_screenshot-crop_guided-linux.png'
+
+import journeyGuidedmacOS from 'images/infrastructure_screenshot-crop_guided-macOS.png'
+
+import journeyGuidedWindows from 'images/infrastructure_screenshot-crop_guided-windows.png'
+
+import journeyNewDash from 'images/infrastructure_screenshot-crop_journey-network-dash.png'
+
+import journeyMainDash from 'images/infrastructure_screenshot-crop_migration-dashboard.png'
+
+Monitoring your infrastructure can be a seemingly daunting task at first glance. Gathering the data from your servers, databases, cloud platforms, and hosted services is only the first task, and once you've set all that up, you still need to find ways to make sense of it all. 
+
+That's where New Relic comes in. Not only do our guided installs and pre-configured dashboards and alerts help you get up and running quickly, our tools will help you proactively troubleshoot and improve your infrastructure. Whether you want to view one or one-hundred of your hosts, you can do everything you need from one central location without having to 
+
+<img
+    title="infrastructure_dashboards"
+    alt="An image displaying New Relic's infrastructure monitoring dashboard"
+    src={journeyMainDash}
+/>
+
+## Your infrastructure monitoring solution [#infra-solution]
+
+New Relic makes it easy for you to observe your entire infrastructure, whether it runs in the cloud or dedicated hosts. In Datadog, you may be familiar with these capabilities as `agents`, and we call the same infrastructure capabilities `agents` at New Relic as well. Here are a few of the offerings for New Relic infrastructure monitoring:
+
+* Infrastructure agents for Linux, macOS, and Windows operating systems with guided and manual install options.
+* An agent for for [Kubernetes](/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration/) that can be set up in a number of ways.
+* Cloud agents for Amazon, Azure, and Google Cloud Platform that you can install independently of our infrastructure agents.
```

Comment:
On the other hand, love the technical specificity and confidence-building here.

Reply:
Thank you!